docs/en/core/tables.md

```diff
@@ -53,6 +53,8 @@ The table editor needs to be enabled in Preferences, under the `Editor` tab. Whe
 
 > Attention! When you activate the table by clicking inside one of the cells, you are entering the edit mode. To save your changes, make sure to **click outside of the table once you are done**, to exit the editing mode! Only then will the changes you've made to the table be applied to the underlying Markdown table.
 
+> Attention! The table editor currently does not support citations, footnotes, and math formulae. You have to disable the table editor in Preferences when you are using any of these in tables.
```

Comment:
Two things:

1. I would rather add this behind line 50, because it fits there and having two warning icons next to each other looks awful and will be detrimental to the cause of informing people here.
2. The statement to disable the table editor when using some of these is just wrong, because the issue only concerns the *display* of those. It does *not*, however, affect any functionality. Citations, footnotes and math in Tables will be correctly exported. They are just not displayed as of yet. Therefore, **disabling the table editor is not a problem**, it will only be good in case you want to preview those.

Reply:
I just read #3412 and apparently I already implemented rendering, there is just a visual glitch that I have to fix. So "does not support citations" is definitely wrong and has to go.